testlib/src/main/resources/zssdk.py

```
@@ -189,7 +188,8 @@ def _check_params(self):
             if value is not None and isinstance(value, str) and annotation.empty_string is False and len(value) == 0:
                 raise SdkError('invalid parameter[%s], it cannot be an empty string' % param_name)
 
-            if value is not None and (isinstance(value, int) or isinstance(value, long)) \
+            int_types = (int,) if sys.version_info[0] >= 3 else (int, long)
```

Ruff (0.15.7)

[error] Undefined name `long`

(F821)